net/local/local_accept.c

```diff
@@ -99,7 +98,7 @@ int local_accept(FAR struct socket *psock, FAR struct sockaddr *addr,
                  FAR socklen_t *addrlen, FAR struct socket *newsock,
                  int flags)
 {
-  FAR struct local_conn_s *server;
+  FAR struct local_conn_s *server = psock->s_conn;
```

Comment:
Should we include a debug assertion for case where "server" is NULL ?

Reply:
The origin code doesn't use assert, this patch just do the simple style change.

Reply:
Yes, but my question was: is the psock and s_conn already DEBUGASSERTed in high end functions before reaching there? Otherwise we will need it.

Reply:
Of course, local_accept is called through psock->s_sockif.accept. It's impossible come to here if psock is NULL.